src/funding/venmo/config.jsx

```diff
@@ -19,6 +20,25 @@ export function getVenmoConfig() : FundingSourceConfig {
             BUTTON_LAYOUT.VERTICAL
         ],
         
+        eligible: ({ experiment }) => {
```

Comment:
Nice improvement! We have another experiment coming up for paylater so this is great to have experiment-specific eligibility logic live inside that funding source config.